### PR TITLE
api description and example body/schema rework

### PR DESCRIPTION
**Pull request by subtosharki**
## Bug

- [ ] Related issues linked using `closes: #number`

## Feature

- [x] Related issues linked using closes: #944 
- [x] There is only 1 db migration with no breaking changes.

## Translations

- [ ] `.json` files are formatted: `yarn format`
- [ ] Translations are correct
- [ ] New translation? It's been added to `next.config.js`

### CODE REVIEW

**Comment by subtosharki** [comment]
https://github.com/SnailyCAD/snaily-cadv4/issues/944

**Comment by subtosharki** [comment]
- [x] /v1/admin 
- [x] /v1/admin/import/citizens 
- [x] /v1/admin/import/vehicles 
- [x] /v1/admin/import/weapons 
- [ ] 
- [ ] 
- [ ] 
- [ ] 
- [ ] 
- [ ] 
- [ ] 
- [ ] 
- [ ]

**Comment by subtosharki** [comment]
@Dev-CasperTheGhost im going to leave the body/schema rework part to you as you know better what is needed

File: packages/api/src/controllers/admin/AdminController.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import type { GetAdminDashboardData } from "@snailycad/types/api";
 @UseBeforeEach(IsAuth)
 export class AdminController {
   @Get("/")
+  @Description("Get all the users within the CAD")
```
Comment (Dev-CasperTheGhost) [inline packages/api/src/controllers/admin/AdminController.ts:18]:
```suggestion
  @Description("Get simple CAD stats")
```

File: packages/api/src/controllers/admin/import/ImportCitizensController.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ import type * as APITypes from "@snailycad/types/api";
 @Controller("/admin/import/citizens")
 export class ImportCitizensController {
   @Post("/")
+  @Description("Import Citizens in the CAD via file upload")
```
Comment (Dev-CasperTheGhost) [inline packages/api/src/controllers/admin/import/ImportCitizensController.ts:18]:
```suggestion
  @Description("Import citizens in the CAD via file upload")
```

minor casing typo

File: packages/api/src/controllers/admin/import/ImportVehiclesController.ts [hunk 3]
```diff
@@ -15,7 +15,8 @@ const vehiclesInclude = { ...citizenInclude.vehicles.include, citizen: true };
 
 @Controller("/admin/import/vehicles")
 export class ImportVehiclesController {
-  @Get()
+  @Get("/")
+  @Description("Get All the Vehciles in the CAD")
```
Comment (Dev-CasperTheGhost) [inline packages/api/src/controllers/admin/import/ImportVehiclesController.ts:19]:
```suggestion
  @Description("Get all the vehicles in the CAD (paginated)")
```

File: packages/api/src/controllers/admin/import/ImportVehiclesController.ts [hunk 4]
```diff
@@ -46,6 +47,7 @@ export class ImportVehiclesController {
   }
 
   @Post("/")
+  @Description("Import Vehciles in the CAD via file upload")
```
Comment (Dev-CasperTheGhost) [inline packages/api/src/controllers/admin/import/ImportVehiclesController.ts:50]:
```suggestion
  @Description("Import vehicles in the CAD via file upload")
```

minor typo

File: packages/api/src/controllers/admin/import/ImportWeaponsController.ts [hunk 5]
```diff
@@ -15,6 +15,7 @@ const weaponsInclude = { ...citizenInclude.weapons.include, citizen: true };
 @Controller("/admin/import/weapons")
 export class ImportWeaponsController {
   @Get("/")
+  @Description("Get All the Weapons in the CAD")
```
Comment (Dev-CasperTheGhost) [inline packages/api/src/controllers/admin/import/ImportWeaponsController.ts:18]:
```suggestion
  @Description("Get all the Weapons in the CAD (paginated)")
```

File: packages/api/src/controllers/admin/import/ImportWeaponsController.ts [hunk 6]
```diff
@@ -43,6 +44,7 @@ export class ImportWeaponsController {
   }
 
   @Post("/")
+  @Description("Import Vehciles in the CAD via file upload")
```
Comment (Dev-CasperTheGhost) [inline packages/api/src/controllers/admin/import/ImportWeaponsController.ts:47]:
```suggestion
  @Description("Import weapons in the CAD via file upload")
```

should be weapons not vehicles here

File: packages/api/src/controllers/admin/manage/AdminManageCustomFieldsController.ts [hunk 7]
```diff
@@ -38,6 +40,7 @@ export class AdminManageCustomFieldsController {
   }
 
   @Put("/:id")
+  @Description("Update a custom field")
```
Comment (Dev-CasperTheGhost) [inline packages/api/src/controllers/admin/manage/AdminManageCustomFieldsController.ts:43]:
```suggestion
  @Description("Update a custom field by its id")
```

File: packages/api/src/controllers/admin/manage/AdminManageCustomFieldsController.ts [hunk 8]
```diff
@@ -66,6 +69,7 @@ export class AdminManageCustomFieldsController {
   }
 
   @Delete("/:id")
+  @Description("Delete a custom field")
```
Comment (Dev-CasperTheGhost) [inline packages/api/src/controllers/admin/manage/AdminManageCustomFieldsController.ts:72]:
```suggestion
  @Description("Delete a custom field by its id")
```

File: packages/api/src/controllers/admin/manage/AdminManageCustomFieldsController.ts [hunk 9]
```diff
@@ -13,12 +13,14 @@ import { UsePermissions, Permissions } from "middlewares/UsePermissions";
...
 export class AdminManageCustomFieldsController {
   @Get("/")
+  @Description("Get all the custom fields within the CAD")
   async getCustomFields() {
     const fields = await prisma.customField.findMany();
     return fields;
   }
 
   @Post("/")
+  @Description("")
```
Comment (Dev-CasperTheGhost) [inline packages/api/src/controllers/admin/manage/AdminManageCustomFieldsController.ts:23]:
```suggestion
  @Description("Create a new custom field")
```

**Review by Dev-CasperTheGhost — CHANGES_REQUESTED** [review]
Added a few suggestions, will commit these myself here:)

**Review by Dev-CasperTheGhost — APPROVED** [review]

**Pull request merged**